src/autopas/utils/AutoPasConfigurationCommunicator.cpp

```diff
@@ -89,7 +93,7 @@ void generateDistribution(const int numConfigs, const int commSize, const int ra
 
   ConfigurationAndRankIteratorHandler iteratorHandler(containerOptions, finiteCellSizeFactors, traversalOptions,
                                                       loadEstimatorOptions, dataLayoutOptions, newton3Options,
-                                                      numConfigs, commSize);
+                                                      verletRebuildFrequencies.getAll(), numConfigs, commSize);
```

Comment:
Is there a reason why this is not handled like `cellSizeFactors`? If not, please handle it like that